pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -21,9 +21,8 @@ import (
 )
 
 const (
-	AlertmanagersKind   = "Alertmanager"
-	AlertmanagerName    = "alertmanagers"
-	AlertManagerKindKey = "alertmanager"
```

Comment:
not sure why you removed this const?

Reply:
I think this isn't being used anywhere, I just searched again in the codebase at it still seems to be the case.

Reply:
This is part of the API, we can't remove it (e.g. it may be used by external clients).

Reply:
Related to https://github.com/prometheus-operator/prometheus-operator/pull/5772